src/main/java/foxtrot/uniform/charlie/kilo/dao/implementations/MovieSessionDaoImpl.java

```diff
@@ -107,4 +108,13 @@ public void deleteById(Long id) {
             }
         }
     }
+
+    @Override
+    public Optional<MovieSession> get(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            return Optional.of(session.get(MovieSession.class, id));
```

Comment:
what is the result if we try to do `Optional.of(null)`?

Reply:
Salary decreasing